nextcloudappstore/core/models.py

```diff
@@ -70,6 +84,33 @@ def can_update(self, user: User) -> bool:
     def can_delete(self, user: User) -> bool:
         return self.owner == user
 
+    def compatible_releases(self, platform_version):
+        all_releases = self.releases.all()
+        return list(filter(lambda rel: rel.is_compatible(platform_version),
+                           all_releases))
+
+    def latest_releases_by_platform_v(self):
+        """Returns a dict with the latest release for each platform version.
+
+        Ignores nightly releases.
+        """
+
+        all_latest = {}
+
+        def latest_non_nightly(releases):
+            latest = None
+            for release in releases:
+                if (latest is None or release.version > latest.version) \
```

Comment:
BTW using a map to turn it into a list of versions and then using Max function would even be better